openedx_webhooks/info.py

```diff
@@ -103,8 +107,8 @@ def _is_pull_request(pull_request, kind):
         # We don't know this person!
         return False
 
-    person = people[author]
     created_at = parse_date(pull_request["created_at"]).replace(tzinfo=None)
+    person = get_person_time_of_pull_request(people[author], created_at)
     if person.get("expires_on", date.max) <= created_at.date():
```

Comment:
This is another twisty thing: is it OK to check "expires_on" in the past-person? Or should we check it on the current person?

Reply:
The "expires_on" key is not within any of the "before" dictionaries so if a user has "expire_on" set, then that PR will be seen as an OSPR regardless if the user info is updated with the "before" information. Should this ordering still be changed to be make this cleaner?

Reply:
I think if "expires_on" can never be changed by "get_updated_person", then we should check it before we call get_updated_person.